test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -0,0 +1,41 @@
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=16 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=24 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=32 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=39 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=46 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=53 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
+]
+stress_cmd: ["scylla-bench -workload=uniform -mode=read -select-order-by=asc -replication-factor=3 -partition-count=60 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -rows-per-request=2000 -timeout=240s -retry-number=10 -concurrency=400 -duration=44h -connection-count 400"
+]
+
+n_db_nodes: 4
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i4i.4xlarge'
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '001'
+instance_type_loader: 'm5n.2xlarge'
+round_robin: true
+nemesis_interval: 30
+nemesis_during_prepare: false
+
+user_prefix: 'longevity-large-partitions-2d'
+
+space_node_threshold: 644245094
+
+# To validate rows in partitions: collect data about partitions and their rows amount
+# before and after running nemesis and compare it
+validate_partitions: true
+table_name: "scylla_bench.test"
+primary_key_column: "pk"
+
+
+run_full_partition_scan: '{"ks_cf": "scylla_bench.test", "interval": 3600, "pk_name":"pk", "rows_count": 50000, "validate_data": "true"}' # 'ks.cf, interval(sec), partition-key name, number-of-rows-per-partition, validate reversed query output, include data-column or only validate pk + ck'
```

Comment:
Comment this out for the first run.
I would like to see only scylla-bench workload in the first run.

Reply:
Ran in: https://jenkins.scylladb.com/job/scylla-staging/job/Longevity_yaron/job/longevity-large-partition-4days-test-rq2/11/
and failed. (error not found in logs, probably loader crashed again and got disconnected)

Argus: https://argus.scylladb.com/test/1b4175fe-4ef3-494c-959a-127a2f105356/runs?additionalRuns[]=0504defa-786c-4368-b954-f1c0eb4a9b9c